src/chrome/content/rules/Cheezburger.xml

```diff
@@ -63,5 +52,4 @@ Non-2xx HTTP code: http://support.cheezburger.com/ (200) => https://cheezburger.
 
```

Comment:
Please replace both with a simple http-to-https rule.

Reply:
Fixed, thanks!